exchanges/binance/binance.go

```diff
@@ -169,10 +169,17 @@ func (b *Binance) GetHistoricalTrades(symbol string, limit int, fromID int64) ([
 func (b *Binance) GetAggregatedTrades(arg *AggregatedTradeRequestParams) ([]AggregatedTrade, error) {
 	params := url.Values{}
 	params.Set("symbol", arg.Symbol)
-	limitSupported := true
+	// fromId xor start time must be set
+	canBatch := arg.FromID == 0 != arg.StartTime.IsZero()
```

Comment:
![image](https://user-images.githubusercontent.com/31364354/99601029-ad69a980-2a52-11eb-8e6c-6e5658bbe666.png)
had to take a moment to appreciate that